pennylane/noise/noise_model.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Contains class and methods for noise models"""
+
+import inspect
+
+import pennylane as qml
+
+
+class NoiseModel:
```

Comment:
In the [docs](https://xanaduai-pennylane--5684.com.readthedocs.build/en/5684/code/api/pennylane.noise.NoiseModel.html), can we get it to show up as `qml.NoiseModel` rather than `qml.noise.NoiseModel`?